packages/cardano-services/src/Asset/DbSyncAssetProvider/DbSyncAssetProvider.ts

```diff
@@ -133,15 +132,6 @@ export class DbSyncAssetProvider extends DbSyncProvider() implements AssetProvid
     return Promise.all(assetIds.map((_) => getAssetInfo(_)));
   }
 
-  private async loadHistory(assetInfo: Asset.AssetInfo) {
-    assetInfo.history = (
-      await this.#builder.queryMultiAssetHistory(assetInfo.policyId, assetInfo.name)
```

Comment:
Please check if we can remove `queryMultiAssetHistory` too

Reply:
Fixed on [64a8945](https://github.com/input-output-hk/cardano-js-sdk/pull/876/commits/64a8945105e1e63269c757e67d6cadf790c23650)